src/_quarto.yml

```diff
@@ -0,0 +1,255 @@
+project:
+  title: "Stan Documentation"
+  type: website
+  output-dir: _website
+  render:
+    - "index.qmd"
+    - "cmdstan-guide/overview.qmd"
+    - "functions-reference/overview.qmd"
+    - "reference-manual/overview.qmd"
+    - "stan-users-guide/overview.qmd"
+    - "cmdstan-guide/*.Rmd"
+    - "functions-reference/*.Rmd"
+    - "reference-manual/*.Rmd"
+    - "stan-users-guide/*.Rmd"
+    - "!sdlc.Rmd"
+    - "!cmdstan-guide/index.Rmd"
+    - "!functions-reference/index.Rmd"
+    - "!reference-manual/index.Rmd"
+    - "!stan-users-guide/index.Rmd"
+
+bibliography: bibtex/all.bib
+biblio-style: apalike
+syntax-definition: "stan.xml"
+toc: true
+toc-depth: 3
+
+website:
+  favicon: img/logo_tm.svg
```

Comment:
I get a warning that this svg doesn't exist

Reply:
fixed